.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ executors:
   node:
     working_directory: ~/lwc
     docker:
-      - image: cimg/node:16.14.0
+      - image: cimg/node:16.19.0
```

Comment:
Current LTS version is v18.13.0, should we use that instead?

Reply:
We have usually co-ordinated major version upgrades across all repos. 
Also there is some bazel work going on where that project is trying work with nodejs v16.
So for those reasons, this PR will only update to latest nodejs v16